.travis.yml

```diff
@@ -1,14 +1,16 @@
 #Testing
 language: csharp
 solution: DSharpPlus.sln
-mono:
-  - latest
+mono: latest
+dotnet: 1.0.1
```

Comment:
I believe SDK version 1.0.3 is required to build, and on top of that, we target a beta version of .NET Standard, which is not covered by Travis. While this is a good change, I am not sure whether it has a point at the moment.